Demo/Demo/Helpers/OrderRequestHelpers.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+import PayPalCheckout
+
+enum OrderRequestHelpers {
```

Comment:
This doesn't need to be an `enum, since it has no cases

Reply:
doing it as a normal class, it throws this warning:
`Convenience Type Violation: Types used for hosting only static members should be implemented as a caseless enum to avoid instantiation. (convenience_type)`
Seems that caseless enums are a valid thing when dealing with only static methods

Reply:
Ah Ok - didn't realize we had a linter rule for that